### PR TITLE
data:theselaw

### PR DESCRIPTION
**Pull request by theselaw**
added my username.json file

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/2724"></a>

### CODE REVIEW

File: data/theselaw.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+{
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/omosaye-moses-0204b6191",
+      "icon": "FaLinkedIn"
+    },
+{
+      "name": "Follow me on Github",
+      "url": "https://github.com/theselaw",
+      "icon": "FaGithub"
+    },
```
Comment (Pradumnasaraf) [inline data/theselaw.json:32]:
```suggestion
    }
```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey,

Please remove the extra comma `,`

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey,

please revert the changes in `package-lock.json` and `package.json` we don't need them in profile addition

**Comment by eddiejaoude** [comment]
If you have any questions, you can come and ask in the EddieHub Discord http://discord.eddiehub.org

**Comment by EmmaDawsonDev** [comment]
Do you need any help with the requested changes @theselaw ?

**Review by SaraJaoude — APPROVED** [review]
looks good with fix in temporary branch

**Pull request merged**